dipy/tracking/utils.py

```diff
@@ -364,31 +364,41 @@ def seeds_from_mask(mask, density=[1, 1, 1], voxel_size=None, affine=None):
         seed point at the center the voxel ``[i, j, k]`` will be represented as
         ``[x, y, z]`` where ``[x, y, z, 1] == np.dot(affine, [i, j, k , 1])``.
 
+    See Also
+    --------
+    random_seeds_from_mask
+
+    Raises
+    ------
+    ValueError
+        When ``mask`` is not a three-dimensional array
+
     Examples
     --------
     >>> mask = np.zeros((3,3,3), 'bool')
     >>> mask[0,0,0] = 1
-    >>> seeds_from_mask(mask, [1,1,1], [1,1,1])
-    array([[ 0.5,  0.5,  0.5]])
-
-    >>> seeds_from_mask(mask, [1,2,3], [1,1,1])
-    array([[ 0.5       ,  0.25      ,  0.16666667],
-           [ 0.5       ,  0.75      ,  0.16666667],
-           [ 0.5       ,  0.25      ,  0.5       ],
-           [ 0.5       ,  0.75      ,  0.5       ],
-           [ 0.5       ,  0.25      ,  0.83333333],
-           [ 0.5       ,  0.75      ,  0.83333333]])
+    >>> random_seeds_from_mask(mask, seeds_per_voxel=1, voxel_size=[1,1,1])
```

Comment:
Why is the example for `seeds_from_mask` calling `random_seeds_from_mask`? Shouldn't this be in the latter function?